src/components/Post.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { TypeComment } from './Comment/TypeComment';
+import { TypeUser } from './User/TypeUser';
+
+import { User } from './User/User';
+import { CommentList } from './Comment/CommentList';
+
+export const Post = ({ post }) => (
+  <>
+    <p>{post.title}</p>
+    <p>{post.body}</p>
+    <User user={post.user} />
+    <CommentList comment={post.comment} />
```

Comment:
```suggestion
    <CommentList comments={post.comment} />
```
Если это list, то он принимает несколько объектов